javascript/javascript1/week2/homework.js

```diff
@@ -0,0 +1,126 @@
+//Formal fullname
+
+
+function getFullname(firstname,surname,useFormalName = false){ 
+    if (useFormalName){         
+        return `Lord.${firstname} ${surname}`
+    }else{
+       return `${firstname} ${surname}`
+    }
+    } 
+    
+    let firstName1=getFullname('Benjamin','Button');
+    let firstName2=getFullname('Benjamin','Button',true);
+    
+    console.log(firstName1);
+    console.log(firstName2);
+
+
+
+
+//Event application 
+
+let today=new Date();
+let todayIndexWeekday=today.getDay();
+const weekdays=[
+    'Sunday',
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    'Saturday']
+
+let eventdaysleft=10;
+let eventday = (todayIndexWeekday+ eventdaysleft) % 7
+console.log(weekdays[eventday]);
```

Comment:
It would have been if you'd done this entire code as a function so it could easily be reused.